### PR TITLE
Added a new tutorial of deploying local stack

### PR DESCRIPTION
**Pull request by yongyanrao**
The `README.md` file includes more information than needed for local funcx deployment, which brings some difficulties to follow for the first time.  Updated the `README.md` file.

### CODE REVIEW

File: local_stack_deployment.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  enabled: false
+funcx_endpoint:
+  image:
+    tag: exception
+
+forwarder:
+  enabled: true
+  tag: main
+  pullPolicy: Always
+  local_image: funcx-forwarder-dev1:latest
```
Comment (yadudoc) [inline local_stack_deployment.md:29]: We should set this something public instead, which can be pulled. For this to work, the user needs to first build a local container image of this name.

**Review by yadudoc — COMMENTED** [review]
I think this text looks mostly reasonable, but we do not want to have a separate additional document and would be better served by updating the current readme with this updated info.

File: README.md [hunk 2]
```diff
@@ -7,154 +7,38 @@ Helm Chart for Deploying funcX stack
...
-secret named `funcx-sdk-tokens`.
-
-You can install this secret with:
-```shell script
-pushd ~/.funcx
-kubectl create secret generic funcx-sdk-tokens --from-file=credentials/funcx_sdk_tokens.json
-popd
-```
-
+2. funcx-endpoint deployed within k8s (broken currently)
```
Comment (yadudoc) [inline README.md:20]: Update this to include the info for internal deployment. This should be working now.

File: README.md [hunk 3]
```diff
@@ -7,154 +7,38 @@ Helm Chart for Deploying funcX stack
...
 
-> :warning: *Only for debugging*: You can set the forwarder curve server key manually by creating
-  a public/secret curve pair, registering them with kubernetes as a secret and then specifying
-  `forwarder.server_cert = true`. By default the forwarder auto generates keys, and distributes it
-  to the endpoint during registration.
+> :warning: **THIS IS A TEST DEPLOYMENT**: The web-service and forwarder are deployed within k8s, while the funcx-endpoint is not yet properly supported.
```
Comment (yadudoc) [inline README.md:22]: We should remove this warning.

**Comment by ZhuozhaoLi** [comment]
Maybe also update the default values in the table at the end.

**Comment by yadudoc** [comment]
These changes are outdated now, and at this point I don't see much value in keeping this open.

**Pull request closed without merging**